src/pages/card/edit/model.ts

```diff
@@ -0,0 +1,28 @@
+import { StartParams } from '@box/lib/page-routing';
+import { attach, createEvent, merge, sample } from 'effector-root';
+import { cardActionsModel } from '@box/features/card/actions';
+import { cardModel } from '@box/entities/card';
+import { historyPush } from '@box/entities/navigation';
+
+import { paths } from '../../paths';
```

Comment:
```suggestion
import { paths } from '@box/pages/paths';
```

nitpick: Кажется лучше использовать абсолютный путь

Reply:
Хз вот, это же +- внутренний ресурс)))